package.json

```diff
@@ -14,27 +14,32 @@
     "build": "node scripts/build.js"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "7.13.14",
-    "@babel/eslint-plugin": "7.13.16",
-    "@babel/plugin-syntax-flow": "7.12.13",
-    "@babel/plugin-syntax-jsx": "7.12.13",
-    "@typescript-eslint/eslint-plugin": "4.22.0",
-    "@typescript-eslint/parser": "4.22.0",
+    "@babel/eslint-parser": "^7.16.3",
+    "@babel/eslint-plugin": "^7.14.5",
+    "@babel/plugin-syntax-flow": "^7.16.0",
+    "@babel/plugin-syntax-jsx": "^7.16.0",
+    "@typescript-eslint/eslint-plugin": "^5.3.1",
+    "@typescript-eslint/parser": "^5.3.1",
     "doctoc": "2.0.0",
-    "eslint": "7.25.0",
+    "eslint": "^8.2.0",
```

Comment:
If you stick to ESLint v7 in your dev dependencies, this PR could be merged without problems since the main changes for adding ESLint v8 compatibility are already done in this PR. Upgrading ESLint here is just for the local environment used to develop the project, that must not block this PR.

Reply:
The package is still compatible with ESLint 7 (see tests), so it can still be merged even though we update the ESLint version for local development

Reply:
But there are some other ESLint plugins not compatible with v8 that can cause errors, that's why I'm suggesting not upgrading to v8 in dev dependencies so it doesn't block the release of the v8 compatibility 😅

Reply:
> block the release of the v8 compatibility

eslint-config-prettier already is (and always has been) compatible with ESLint 8. This PR is just adding tests for ESLint 8 and upgrading to using ESLint 8 internally. There’s no rush with this PR.

Marking as “unresolved” to increase visibility of this comment.